src/core/etl/tests/Flow/ETL/Tests/Integration/DataFrame/PartitioningTest.php

```diff
@@ -163,13 +163,13 @@ public function test_partition_by_partitions_order() : void
                     function (int $i) : array {
                         $data = [];
 
-                        $maxItems = \random_int(2, 10);
```

Comment:
More I look at this, more I think I would add new function for unique int/string to "hide" object call & improve DX perspective, WDYT?